pyro/contrib/gp/models/model.py

```diff
@@ -1,87 +1,193 @@
 from __future__ import absolute_import, division, print_function
 
 from pyro.contrib.gp.util import Parameterized
-from pyro.infer import SVI
+from pyro.infer.svi import SVI
 from pyro.optim import Adam, PyroOptim
 
 
 class GPModel(Parameterized):
     """
-    Base class for models used in Gaussian Process.
-
-    :param torch.Tensor X: A 1D or 2D tensor of input data for training.
-    :param torch.Tensor y: A tensor of output data for training with
-        ``y.shape[-1]`` equals to number of data points.
-    :param pyro.contrib.gp.kernels.Kernel kernel: A Pyro kernel object.
-    :param torch.Size latent_shape: Shape for latent processes. By default, it equals
-        to output batch shape ``y.shape[:-1]``. For the multi-class classification
-        problems, ``latent_shape[-1]`` should corresponse to the number of classes.
-    :param float jitter: An additional jitter to help stablize Cholesky decomposition.
+    Base class for Gaussian Process models.
+
+    The core of a Gaussian Process is a covariance function :math:`k` which governs
```

Comment:
This is a great docstring, it seems wasted on `GPModel` which is not user-facing.  Is there somewhere more prominent you could put it?

Reply:
I think that we move it to GPRegression and make a note which link to it for other models. What's your opinion about that?

Reply:
Maybe leave it in GPModel in its entirety, and also copy the first section (lines 10-23) into a notes section at the end of every GPModel subclass?